crates/ef-testing/tests/tests.rs

```diff
@@ -120,9 +120,9 @@ mod blockchain_tests {
     blockchain_tests!(st_sstore_test, stSStoreTest);
     blockchain_tests!(st_stack_tests, stStackTests);
     blockchain_tests!(st_static_call, stStaticCall);
+    blockchain_tests!(st_time_consuming, stTimeConsuming);
```

Comment:
Could you move this back so we have the tests in the same order as they appear on the GeneralTests folder?

Reply:
done